mmaction/datasets/pipelines/augmentations.py

```diff
@@ -42,6 +43,39 @@ def _init_lazy_if_proper(results, lazy):
         assert 'lazy' not in results, 'Use Fuse after lazy operations'
 
 
+@PIPELINES.register_module()
+class EntityBoxRescale:
+
+    def __init__(self, scale_factor):
+        warnings.warn('"EntityBoxRescale" is deprecated and will be removed '
```

Comment:
Make this an error, since the code will not behave correctly.

Reply:
And provide this pr link in the error message. `"Details see https://github.com/open-mmlab/mmaction2/pull/782"`

Reply:
Actually, this module is never used in the data pipeline, and 
**it never behaves correctly in the data pipeline (when used alone)**. 
I think no one has used it in the data pipeline.